system-test/storage.ts

```diff
@@ -2809,6 +2809,7 @@ describe('storage', () => {
     it('should create a signed read url', done => {
       file.getSignedUrl(
           {
+            version: 'v2',
```

Comment:
Is `v2` still the default if none is passed?

Reply:
Yes, with the expectation `v4` will be made the default in the future.